code/__DEFINES/job.dm

```diff
@@ -8,7 +8,7 @@
 #define SQUAD_MARINE_3					"Charlie"
 #define SQUAD_MARINE_4					"Delta"
 #define SQUAD_MARINE_5					"Echo"
-#define SQUAD_MARINE_CRYO				"Reserves"
+#define SQUAD_MARINE_CRYO				"Foxtrot"
```

Comment:
IMO this change just makes the reserves squad more confusing 
it should still be called reserves
or just put the cryorines into random squads